worker/interfaces.go

```diff
@@ -1,16 +1,9 @@
 package worker
 
 import (
-	"context"
 	"github.com/ohsu-comp-bio/funnel/proto/tes"
 )
 
-// Worker is a type which runs a task.
-type Worker interface {
```

Comment:
Why was this interface deleted?

Reply:
It's only needed by the node, so it lives there now, and replaced the WorkerFactory.